pkg/rtc/uptrackmanager.go

```diff
@@ -60,7 +60,7 @@ func (u *UpTrackManager) Start() {
 
 func (u *UpTrackManager) Restart() {
 	for _, t := range u.GetPublishedTracks() {
-		t.Restart()
+		t.(types.LocalMediaTrack).Restart()
```

Comment:
would there be potentially casting errors? i.e. a regular MediaTrack in the list of published tracks?

Reply:
I think this can be moved to `LocalParticipant` (where tracks will be of type `LocalMediaTrack` so the casting will not look odd). Will move it.